src/core/mac/mac_frame.hpp

```diff
@@ -95,6 +95,21 @@ class ExtAddress : public otExtAddress
      */
     typedef String<kInfoStringSize> InfoString;
 
+    /**
+     * This method generates a random IEEE 802.15.4 Extended Address with Group bit not set and Local bit set.
+     *
+     */
+    void GenerateRandom(void) { GenerateRandom(/* aGroup */ false, /* aLocal */ true); }
+
+    /**
+     * This method generates a random IEEE 802.15.4 Extended Address with a given Group bit and Local bit.
+     *
+     * @param[in]  aGroup  TRUE to set the Group bit, FALSE to clear it.
+     * @param[in]  aLocal  TRUE to set the Local bit, FALSE to clear it.
+     *
+     */
+    void GenerateRandom(bool aGroup, bool aLocal);
```

Comment:
I think we can remove `aGroup` and `aLocal` parameters - these should always be false and true, respectively.

Reply:
Done!